README.md

```diff
@@ -1,117 +1,100 @@
-shiny.telemetry
-============
+# shiny.telemetry
 
-Easy way for logging users activity and adding statistics panel to your Shiny app.
+> Easy way for logging users activity and adding statistics panel to your Shiny app
 
-## Prerequisites
+## Install
 
-Remote DB. Can be PostgreSQL DB or simple sqlite file.
-
-## How to register user's action?
-
-1. Prepare DB to store results (here we use SQLite, but PostgreSQL is possible as well).
+Shiny.telemetry can be installed from github by using the remotes package:
 
 ```
-# run only once
-connection <- DBI::dbConnect(RSQLite::SQLite(), dbname = "user_stats.sqlite")
-DBI::dbDisconnect(connection)
+remotes::install_github("Appsilon/shiny.telemetry")
 ```
 
-2. Depending on your authentication approach, define function to extract username.
-In this example we just extract username from url `username` parameter.
+## How to use in  a Shiny Dashboard?
+
+This package can be used with a minimal setup that keeps track of. 
```

Comment:
Oh no. I wanted to update the readme a bit so it starts to reflect the new version os `shiny.telemetry`, but got lost while doing so.

It will require an issue on its own, but for these unfinished sentences I'll correct them in this PR

Reply:
README was re-written